Library/Homebrew/utils/repology.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require "net/http"
+require "json"
+
+module RepologyParser
+  def call_api(url)
+    puts "- Calling API #{url}"
+    uri = URI(url)
+    response = Net::HTTP.get(uri)
+
+    puts "- Parsing response"
+    JSON.parse(response)
+  end
+
+  def query_repology_api(last_package_in_response = "")
+    url = "https://repology.org/api/v1/projects/#{last_package_in_response}?inrepo=homebrew&outdated=1"
+
+    call_api(url)
+  end
+
+  def parse_repology_api
+    puts "\n-------- Query outdated packages from Repology --------"
+    page_no = 1
+    puts "\n- Paginating repology api page: #{page_no}"
+
+    outdated_packages = query_repology_api("")
```

Comment:
Why `""`?

Reply:
@MikeMcQuaid this is because of the way Repology paginates it's API

Reply:
Sorry: I mean given it's the default argument: why pass it here?

Reply:
I'm removing this.  Thanks for pointing it out, Mike!